qa/workunits/rbd/cli_migration.sh

```diff
@@ -75,13 +75,17 @@ test_import_native_format() {
     local base_image=$1
     local dest_image=$2
 
+    rbd migration prepare --import-only "rbd/${base_image}@2" ${dest_image}
+    rbd migration abort ${dest_image}
+
     local pool_id=$(ceph osd pool ls detail --format xml | xmlstarlet sel -t -v "//pools/pool[pool_name='rbd']/pool_id")
     cat > ${TEMPDIR}/spec.json <<EOF
 {
   "type": "native",
   "pool_id": ${pool_id},
   "pool_namespace": "",
-  "image_name": "${base_image}"
+  "image_name": "${base_image}",
+  "snap_name": "2"
```

Comment:
@dillaman need to remove `compare_images "${base_image}" "${dest_image}"` below, as we do not migrate the head now. The current test fails [1].

[1] http://qa-proxy.ceph.com/teuthology/trociny-2020-12-29_08:24:55-rbd-wip-mgolub-testing-distro-basic-smithi/5746655/teuthology.log